include/deal.II/base/symmetric_tensor.h

```diff
@@ -21,7 +21,6 @@
 #include <deal.II/base/numbers.h>
 #include <deal.II/base/table_indices.h>
 #include <deal.II/base/template_constraints.h>
-#include <deal.II/base/vectorization.h>
```

Comment:
hm, this was added in https://github.com/dealii/dealii/pull/3967 do the test still work for you?